bullet-journal/src/Components/Journal/VerticalLayout/DayView.js

```diff
@@ -1,68 +1,158 @@
-import React, {useState, Fragment} from 'react'
-import Bullet from './Bullet'
-
-function DayView(props) {
-
-    const mockBullets = [
-        {
-            date: '07/06/2020',
-            category: 'Work', 
-            text: 'Work on Bullet Journal'
-        },
-        {
-            date: '07/06/2020',
-            category: 'Work',
-            text: '3 easy coding problems'
-        },
-        {
-            date: '07/06/2020',
-            category: 'Casual',
-            text: 'Stream on Twitch'
-        },
-        {
-            date: '07/06/2020',
-            category: 'Casual',
-            text: 'Practice piano'
-        }
-    ]
-    
-    const [bullets, setBullets] = useState(mockBullets);
-    const [day, setDay] = useState({
-        name: 'Wednesday',
-        date: '07/01/2020'
-    })
+import React, {Fragment, useReducer, useRef} from 'react'
+import {v4 as uuid} from 'uuid'
+import Bullet from './Bullet';
+
+export const DayBulletsContext = React.createContext();
+
+const mockBullets = [
```

Comment:
It's ok to use fake data, but make sure you give the user a way to add new data or clear out the fake data.